vms/proposervm/post_fork_block.go

```diff
@@ -17,6 +17,10 @@ var _ PostForkBlock = (*postForkBlock)(nil)
 type postForkBlock struct {
 	block.SignedBlock
 	postForkCommonComponents
+
+	// slot is used to store the selected slot for this block. It's being used during Accept, and populated
+	// during verifyPostDurangoBlockDelay.
```

Comment:
Optional: I think this reads a bit better.
```suggestion
	// slot of the proposer that produced this block.
	// It is populated in verifyPostDurangoBlockDelay.
	// It is used to report metrics during Accept.
```